src/rapid_pro_to_engagement_db/cache.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+import json
+
+from core_data_modules.util import IOUtils
+from temba_client.v2 import Contact
+
+
+class RapidProSyncCache(object):
```

Comment:
Since we're using python 3.x, do we need to inherit from object? Anyway I know most of our class inherit from object, if its for consistency I think you can just leave as is.

Reply:
No we don't, this was just habit. I will remove this here and try to stop using it in future, that way we need to understand one fewer of python's mistakes going forwards.